docker/jupyterlab/env.sh

```diff
@@ -58,3 +58,4 @@ start_ssh_agent
 init_ssh_agent_env
 EOF
 chown -R jovyan "$HOME/.local/profile.skel"
+chmod 0700 /home/jovyan/.cache//git/credential
```

Comment:
This command will fail if` /home/jovyan/.cache/git/credential `doesn't exist. Can you wrap it in an if-statement or something?

Reply:
Even better, use the flag `--silent`

Reply:
done